static/app/components/events/aiAutofix/useAiAutofix.tsx

```diff
@@ -7,10 +7,16 @@ import type {
 import {useApiQuery} from 'sentry/utils/queryClient';
 import useApi from 'sentry/utils/useApi';
 
-const POLL_INTERVAL = 5000;
+const POLL_INTERVAL = 2500;
 
 export const useAiAutofix = (group: GroupWithAutofix) => {
   const api = useApi();
+
+  const [additionalContext, setAdditionalContext] = useState<string>('');
```

Comment:
Don't need the type here since it can be inferred

```suggestion
  const [additionalContext, setAdditionalContext] = useState('');
```